### ISSUE TITLE
y-scale missing in stacked_violin for 1.5.2.dev

### ISSUE THREAD

**Issue by z5ouyang** [scverse/scanpy#1321]
The y-scale label on the right of the stacked_violin is missing in the version of 1.5.2.dev

```python
import scanpy as sc
pbmc = sc.datasets.pbmc68k_reduced()
marker_genes = ['S100A8', 'GNLY', 'NKG7', 'KLRB1', 'FCGR3A', 'FCER1A', 'CST3']
sc.pl.stacked_violin(pbmc,marker_genes,groupby='louvain',row_palette='muted',figsize=(7,4),log=False)
sc.logging.print_versions()
```

![download](https://user-images.githubusercontent.com/30639029/87759504-434ea880-c7c3-11ea-9c3a-b59e027ffeb6.png)

#### Versions:

> scanpy==1.5.2.dev123+g0cf308c anndata==0.7.3 umap==0.4.4 numpy==1.18.5 scipy==1.4.1 pandas==1.0.4 scikit-learn==0.23.1 statsmodels==0.11.1

**Comment by ivirshup** [comment]
@fidelram, could you take a look at this?

**Comment by fidelram** [comment]
Should be possible to turn the y ticks legends on. But I just tested it and didn't work. I will try to fix it.

The syntax is:
```PYTHON
sc.pl.stacked_violin(adata,marker_genes,groupby='louvain', return_fig=True).style(yticklabels=True,row_palette='muted').show()
```

`style` needs to be used to tune the graphical parameters to avoid overcrowding the parameters list. But I am open to have a discussion on what the users think is best.

Documentation is here: https://scanpy.readthedocs.io/en/latest/api/scanpy.pl.DotPlot.style.html#scanpy.pl.DotPlot.style

**Comment by fidelram** [comment]
Solved in #1342
